client/app/views/popover_screens/duplicate.coffee

```diff
@@ -0,0 +1,38 @@
+PopoverScreenView = require 'lib/popover_screen_view'
+
+module.exports = class ConfirmDuplicatePopoverScreen extends PopoverScreenView
```

Comment:
I thought about that but it's not literally a duplicate popover. It's a popover to confirm if the user want to continue or not to duplication and discarding old changes. I know it's long name but maybe more explicit here. What do you think?